Reflection_Engine/Modify/SetPropertyValue.cs

```diff
@@ -93,6 +93,12 @@ public static bool SetPropertyValue(this object obj, string propName, object val
                 bhomObj.CustomData[propName] = value;
                 return true;
             }
+            else if (obj is IDictionary)
+            {
+                IDictionary dic = obj as IDictionary;
+                dic[propName] = value;
```

Comment:
What happens if the dictionary does not contain a key for `propName`? I'd like a check on it as a safety precaution please :smile:

Reply:
Key and value are added to CustomData accordingly.

![image](https://user-images.githubusercontent.com/6752345/76662595-ad1c2480-653b-11ea-87dc-9cee8c04bd31.png)

Reply:
@FraserGreenroyd , long day ? 😉

Reply:
@adecler long week 😿 - marking as resolved cause this isn't relevant 🤦‍♂